Topic6/src/main/java/com/aguirre/topic6/Meeting.java

```diff
@@ -0,0 +1,49 @@
+package com.aguirre.topic6;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+public class Meeting {
+
+	@Id
+	@GeneratedValue
+	private Long id;
+	private String meetingName;
+	private List<Attendee> attendees;
+	private Room room;
+	private int timeSlot;
+
+	public Meeting(String meetingName, List<Attendee> attendees, Room room, int timeSlot) {
+		update(meetingName, attendees, room, timeSlot);

```

Comment:
It's not clear to use the update method here only to re-use it, in this method you should use simply:
this.meetingName = meetingName;
this.attendees = attendees;
this.room = room;
this.timeSlot = timeSlot;